### ISSUE TITLE
[Forgot your password? - Request Account] Error after entering username

### ISSUE THREAD

**Issue by kchatpar** [aces/Loris#4543]
When a user submits their username to reset their password on this page:
https://test.loris.ca/login/password-reset/

Instead of getting the notice that an email was sent an error is thrown and no email is sent.

**Comment by johnsaigle** [comment]
Maybe related to https://github.com/aces/Loris/issues/4515. Email doesn't appear to work on the server.

**Comment by driusan** [comment]
What's the error that's thrown?

**Comment by ridz1208** [comment]
```
[Tue May 14 09:23:08.337245 2019] [php7:notice] [pid 26039] [client 132.216.56.93:33748] PHP Notice:  Undefined index: Email in /var/www/loris/php/libraries/User.class.inc on line 213, referer: https://test.loris.ca/login/password-reset/
[Tue May 14 09:25:08.582713 2019] [php7:error] [pid 26038] [client 132.216.56.93:33796] PHP Fatal error:  Uncaught TypeError: uniqid() expects parameter 1 to be string, integer given in /var/www/loris/php/libraries/Email.class.inc:78\nStack trace:\n#0 /var/www/loris/php/libraries/Email.class.inc(78): uniqid(1557840308)\n#1 /var/www/loris/modules/login/php/passwordreset.class.inc(80): Email::send('rida.loris@gmai...', 'lost_password.t...', Array)\n#2 /var/www/loris/php/libraries/NDB_Form.class.inc(162): LORIS\\login\\PasswordReset->_process(Array)\n#3 /var/www/loris/php/libraries/LorisForm.class.inc(1438): NDB_Form->_save(Array)\n#4 /var/www/loris/php/libraries/NDB_Form.class.inc(133): LorisForm->process(Array)\n#5 /var/www/loris/php/libraries/NDB_Form.class.inc(190): NDB_Form->save()\n#6 /var/www/loris/src/Middleware/AnonymousPageDecorationMiddleware.php(55): NDB_Form->handle(Object(Zend\\Diactoros\\ServerRequest))\n#7 /var/www/loris/src/Middleware/PageDecorationMiddleware.php(38): LORIS\\Middleware\\AnonymousPageDecorationMiddleware->process(Object(Zend\\Diactoros\\ServerRequest), Object(LORIS\\login\\PasswordReset))\n#8 /var/www/l in /var/www/loris/php/libraries/Email.class.inc on line 78, referer: https://test.loris.ca/login/password-reset/
```

This is the second module throwing an error because of the email.

user_accounts has the same problem

**Comment by johnsaigle** [comment]
@driusan @ridz1208 This will be resolved by addressing #4561. The Test VM needs the latest code from major. This should resolve the email issue people are seeing.

**Comment by zaliqarosli** [comment]
fixed on test.loris.ca
